### PR TITLE
fix : returned shapeId by addShape()

### PR DESCRIPTION
**Pull request by jbousquie**
the returned shapeId by addShape() was wrong.

### CODE REVIEW

File: src/Particles/babylon.solidParticleSystem.ts [hunk 1]
```diff
@@ -286,7 +286,7 @@ module BABYLON {
             }
             this.nbParticles += nb;
             this._shapeCounter++;
-            return this._shapeCounter;
+            return this._shapeCounter - 1;
```
Comment (RaananW) [inline src/Particles/babylon.solidParticleSystem.ts]: `return this,_shapeCounter++`? :smile:

**Comment by jbousquie** [comment]
No, we must store shapeCounter incremented, but we must return the value before incrementation

**Comment by RaananW** [comment]
that's exactly what this line will do (put a return in line 288, remove the rest). return the original value, and increment right after. But it is not too readable. The way you did it is clearer.

**Comment by jbousquie** [comment]
ok, I thought it would increment first, then return the computed value

**Comment by Temechon** [comment]
@jbousquie : 
- increment first, then return : ++this._shapeCounter
- return, then increment: this._shapeCounter++

**Pull request merged**